src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/Android/MediaElementRenderer.android.cs

```diff
@@ -375,11 +375,18 @@ protected void UpdateVolume()
 
 			mediaPlayer?.SetVolume((float)MediaElement.Volume, (float)MediaElement.Volume);
 		}
+
 		protected void UpdateSpeed()
 		{
 			if (MediaElement == null || mediaPlayer == null)
 				return;
 
+			if (Helpers.XCT.SdkInt < 23)
+			{
+				Console.WriteLine("MediaElement Speed control functionality is not available. Minimum supported API is 23");
```

Comment:
I would say that we should throw an exception, to be _very_ clear to the dev that this feature isn't supported in old android versions. What do you think?

Reply:
If we wanted to do that we may want to rework some bits then as this `UpdateSpeed` method is called once the media starts playing. In fact I best double check this! I *think* it was called when starting the video playback

This now allows for the video to play correctly the user just can't change the speed.

Reply:
I have dug a little deeper in to this, if I remove the calling for `UpdateSpeed` it still behaves (at least on Android 5). This means we could throw a `NotSupportedException` when the user tries to change the speed, it does seem to wrap that exception inside a `System.Reflection.TargetInvocationException` though. Would this still be fine?

If we do this I would suggest that we also add a property like `IsSpeedSupported` so that the developer can choose whether to allow their users to change the speed in their apps. What do you think?

Reply:
Adding a new prop is a change in our api and we don't want that. So the solution on this PR is the best one. We can create this new prop on mct. 
Also, can we change the color of that message on console? This will highlight it for devs

Reply:
`Trace.WriteLine` provides some better color. Do you think this is enough?

![Screenshot 2022-02-16 at 21 40 25](https://user-images.githubusercontent.com/17139988/154362080-4a5eb7bd-7d4f-4332-8473-fb04deb5fadc.png)

Reply:
Yeap, you rock <3